lib/experimental/fonts/font-library/class-wp-font-collection.php

```diff
@@ -21,41 +21,128 @@
 class WP_Font_Collection {
 
 	/**
-	 * Font collection configuration.
+	 * The unique slug for the font collection.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @var string
+	 */
+	private $slug;
```

Comment:
Browsing through PHP classes in `wp-includes/`, I see a lot with public properties (e.g. WP_Post, WP_Term, etc). What to you think of making these properties public, instead? (at least for: slug, name, description, and categories)

Reply:
Done